### PR TITLE
Improvements for security review #2

### PR DESCRIPTION
**Pull request by rianvdm**
@temochka Can you check if this looks ok?

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -298,17 +299,12 @@ app.post('/command/postmark', function(req, res) {
...
-    request({
-      url: slashResponseURL,
-      method: 'POST',
-      json: true,
-      body: {
-        "response_type": "ephemeral",
-        "text": 'Sorry, it looks like this request didn\'t originate from Slack.',
-        },
-      }, function(error, response, body) {});
+    res.status(403).end();
```
Comment (temochka) [inline app.js]: If I read this correctly, the response has already been sent (line 294). You might want to move that line down below if you want to retain the ability to respond with a different status.

File: app.js [hunk 2]
```diff
@@ -298,17 +299,12 @@ app.post('/command/postmark', function(req, res) {
   
   // Validate the request is legit
   
-  if (slashToken !== process.env.SLASH_TOKEN) {
+  var a = Buffer.alloc(24, slashToken);
```
Comment (temochka) [inline app.js]: Seems like it’d be more practical to determine the buffer size dynamically based on the length of `process.env.SLASH_TOKEN`. This way you’d avoid a hard-coded constant and your code would still work even if Slack changed the length of their tokens.

File: app.js (hunk 1 above)
Comment (rianvdm) [reply app.js]: Yeah that's a good point - moving.

File: app.js (hunk 2 above)
Comment (rianvdm) [reply app.js]: Yep on it.

**Comment by rianvdm** [comment]
@temochka Ok cool - check it now :)

**Pull request merged**